src/index.ts

```diff
@@ -8,39 +8,27 @@ import Crypto from './crypto'
 
 /**
  * Entrypoint into the FormSG SDK
- * @param {PackageInitParams} config Package initialization config parameters
- * @param {string} [config.publicKey] This public key is used to authenticate or verify signed objects passed into this package.
- * @param {string?} [config.webhookSecretKey] Optional base64 secret key for signing webhooks
- *//**
- * Deprecated entrypoint into the FormSG SDK
  *
- * @deprecated since January 2021
  * @param {PackageInitParams} config Package initialization config parameters
- * @param {string?} [config.publicKey] Optional. If provided, this public key will be used instead to authenticate or verify signed objects passed into this package.
  * @param {string?} [config.mode] Optional. Initializes public key used for verifying and decrypting in this package. If `config.signingPublicKey` is given, this param will be ignored.
- * @param {string?} [config.webhookSecretKey] Optional base64 secret key for signing webhooks
+ * @param {string?} [config.webhookSecretKey] Optional. base64 secret key for signing webhooks. If provided, enables generating signature and headers to authenticate webhook data.
+ * @param {VerificationOptions?} [config.verificationOptions] Optional. If provided, enables the usage of the verification module.
  */
 export = function (config: PackageInitParams = {}) {
-  const { publicKey, webhookSecretKey, mode, verificationOptions } = config
-  const encryptionPublicKey =
-    publicKey || getSigningPublicKey(mode || 'production')
-  const verificationPublicKey =
-    publicKey || getVerificationPublicKey(mode || 'production')
-
-  if (typeof mode !== 'undefined') {
-    console.warn('Initializing the FormSG SDK with the `mode` parameter is now deprecated. Please switch to the publicKey parameter instead.')
-  }
+  const { webhookSecretKey, mode, verificationOptions } = config
+  const signingPublicKey = getSigningPublicKey(mode || 'production')
+  const verificationPublicKey = getVerificationPublicKey(mode || 'production')
```

Comment:
i think we could use some comments to remind ourselves what the purpose of these are for

Reply:
fixed in 9b6b243